mnt/reform2-nitrogen8m/default.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+{
+  boot = {
+    initrd = {
+      availableKernelModules = [ "nvme" "usbhid" ];
+      kernelModules = [ "nwl-dsi" "imx-dcss" ];
+    };
+    extraModprobeConfig = "options imx-dcss dcss_use_hdmi=0";
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_reformNitrogen8m_latest;
+    kernelParams =
+      [ "console=ttymxc0,115200" "console=tty1" "cma=512M" "pci=nomsi" ];
```

Comment:
I wonder if, just as in [#97883](https://github.com/NixOS/nixpkgs/pull/97883), we shouldn't set `cma=` on the command-line.

 * https://github.com/NixOS/nixpkgs/pull/97883/commits/3a13b44e862c0a6b86de975aa1a0c9dea1e4adc9

The custom kernel config is already setting it. So I don't think it's needed here.

* * *

Though, as our default universal kernel config sets it as 32M, an hypothetical upcoming time where this computer uses mainline with defconfig would need to see this set in some ways.

I would tend to think that it would be better off as a default `dtbo` that's shipped with the proper configuration for this device, in a future update. I mean, I don't even know myself how to do it exactly, so I wouldn't ask someone to do the dirty work of figuring this out. You're just caught in the cross-fire between my neurons.